get-started/index.md

```diff
@@ -335,9 +335,9 @@ COPY target/dependency/cryostat-agent.jar /deployments/app/
 ENV JAVA_OPTS="-javaagent:/deployments/app/cryostat-agent.jar"
 ```
 
-Next we must rebuild our container image. This is specific to your application but will likely look something like `docker build -t docker.io/myorg/myapp:latest -f src/main/docker/Dockerfile .`.
-Push that updated image or otherwise get it updated in your Kubernetes registry, then modify your application `Deployment` to supply JVM system properties or environment variables configuring
-the Cryostat Agent:
+Next we <span style="color:red">must</span> rebuild our container image. This is specific to your application but will likely look something like `docker build -t docker.io/myorg/myapp:latest -f src/main/docker/Dockerfile .`.
```

Comment:
I don't think "must" needs to be made red here. This isn't a critical warning, it's just the next step in this procedure.